tests/unit/utils/test_extend_pml.py

```
@@ -0,0 +1,256 @@
+        inv_perm = jnp.ones((1, 10, 5, 5))
+        inv_perm = inv_perm.at[:, 2, :, :].set(0.5)
+        arrays = SimpleContainer(
+            inv_permittivities=inv_perm,
+            inv_permeabilities=1.0,
+            electric_conductivity=None,
+            magnetic_conductivity=None,
+        )
+        return objects, arrays
+
+    def test_minus_direction_basic(self):
+        """PML cells (indices 0–1) should be filled with the interior-edge value 0.5."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)